packages/components/src/date-time/date/test/index.tsx

```diff
@@ -47,7 +43,7 @@ describe( 'DatePicker', () => {
 		);
 
 		await user.click(
-			screen.getByRole( 'button', { name: 'Friday, May 20, 2022' } )
+			screen.getByRole( 'button', { name: 'May 20, 2022' } )
```

Comment:
Yes! Using the format string in `wp.date.__experimentalGetSettings().formats.date` instead of a hardcoded format string ensures that the ARIA label respects the site's _Date Format_ setting:

<img width="498" alt="Screen Shot 2022-08-23 at 10 07 00" src="https://user-images.githubusercontent.com/612155/186040747-6bd16583-3831-4af7-8dc3-4b4556e65113.png">

It also ensures that the format string is internationalised because the format string obtained from `wp-date` has been passed through `__`:

https://github.com/WordPress/wordpress-develop/blob/85ae9dd89b4c91f61a3517f7c41813f53cde87fd/src/wp-includes/script-loader.php#L412

This is important as not all languages use `{Day}, {Month} {Day}, {Year}`.

Worth noting that prior to this PR these labels weren't working anyway. The labels contained format string tokens:

<img width="309" alt="Screen Shot 2022-08-23 at 10 00 10" src="https://user-images.githubusercontent.com/612155/186040176-2e132f4c-6c39-470d-8bd4-a949a6dc459b.png">

Reply:
Sounds good, thank you for the context!


_"Are you free on Thursday F, j?"_